pkg/apis/authentication/validation/validation.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	authenticationapi "k8s.io/kubernetes/pkg/apis/authentication"
+)
+
+
+// ValidateTokenReviewSpec check the validation of TokenReviewSpec field.
+func ValidateTokenReviewSpec(spec authenticationapi.TokenReviewSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if len(spec.Token) == 0 {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("token"), spec.Token, `must be specified`))
```

Comment:
Yes, we can make use of field.Required. Thanks for your reminding, that's a nice help.